func_sig_registry/registry/forms.py

```diff
@@ -41,6 +51,37 @@ def validate_text_signature(self, value):
         return normalize_function_signature(value)
 
 
+class AllSignatureCreateForm(serializers.Serializer):
+    function_text_signature = serializers.CharField(read_only=True)
+    event_text_signature = serializers.CharField(read_only=True)
+
+    def create(self, validated_data):
+        message = ''
+        if 'function_text_signature' in validated_data:
+            function_signature = Signature.import_from_raw_text_signature(
+                validated_data['function_text_signature'],
+                )
+            message += 'Added function signature {0} for function {1}'.format(
+                function_signature.bytes_signature.get_hex_display(),
+                function_signature.text_signature,
+            )
+        if 'event_text_signature' in validated_data:
+            event_signature = EventSignature.import_from_raw_text_signature(
+                validated_data['event_text_signature'],
+            )
+            message += 'Added event signature {0} for event {1}'.format(
```

Comment:
For message construction here you might do it something like this.

```python
message_parts = []
if condition_a:
    message_parts.append("Did something for condition A")
if condition_b:
    message_parts.append("Did something for condition B")

message = ', '.join(message_parts)
```

This will give you a nice message whether one or both messages are present.